### PR TITLE
add kitty terminal color sheme

### PR DESCRIPTION
**Pull request by theo-grivel**
Create a theme for this terminal emulator (https://sw.kovidgoyal.net/kitty/)

### CODE REVIEW

File: terminal/kitty/cyberpunk-neon.conf [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Cyberpunk Neon: https://github.com/Roboron3042/Cyberpunk-Neon
+# Author: Theo (theo-grivel)
+
+background            #000b1e
+foreground            #0abdc6
+selection_background  #fcf6e8
```
Comment (Roboron3042) [inline terminal/kitty/cyberpunk-neon.conf:6]: Where did this come from?

Comment (theo-grivel) [reply terminal/kitty/cyberpunk-neon.conf:6]: I use this file for copy the colors...
https://github.com/Roboron3042/Cyberpunk-Neon/blob/master/terminal/termite/config

and I use this manual of kitty...
https://sw.kovidgoyal.net/kitty/conf/#color-scheme

Comment (Roboron3042) [reply terminal/kitty/cyberpunk-neon.conf:6]: But the fcf6e8 (selection_background) colour is nowhere in the colour scheme.

Looking at the kitty config, setting both selection_foreground and selection_background to "none" would reverse the colours. Wouldn't this be desirable?

Comment (theo-grivel) [reply terminal/kitty/cyberpunk-neon.conf:6]: Yep, it's clearly better!

**Comment by Roboron3042** [comment]
Thank you very much for your contribution to Cyberpunk-Neon 🙂

**Pull request merged**